istio-operator-1.19.advisories.yaml

```diff
@@ -0,0 +1,21 @@
+schema-version: "2"
+
+package:
+  name: istio-operator-1.19
+
+advisories:
+  - id: CVE-2019-3826
+    events:
+      - timestamp: 2023-09-25T20:27:23Z
+        type: false-positive-determination
+        data:
+          type: vulnerability-record-analysis-contested
+          note: Prometheus version 0.45 is actually 2.45 which is newer than the fixed version 2.7.1
+
+  - id: GHSA-6xv5-86q9-7xr8
+    events:
+      - timestamp: 2023-09-25T20:31:37Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
```

Comment:
I'd change this to `vulnerable-code-not-included-in-package` since the Windows code isn't included in wolfi!

Reply:
Done, thanks